pkg/pb/gracefulexit.proto

```diff
@@ -0,0 +1,77 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+syntax = "proto3";
+option go_package = "pb";
+
+import "gogo.proto";
+import "metainfo.proto";
+import "orders.proto";
+
+package gracefulexit;
+
+service SatelliteGracefulExit {
+  // Process is called by storage nodes to initiate the graceful exit, get pieces to transfer, and receive exit status.
+  rpc Process(stream StorageNodeMessage) returns (stream SatelliteMessage);
+}
+
+message TransferSucceeded {
+  metainfo.AddressedOrderLimit addressed_order_limit   = 1;
+  orders.PieceHash             original_piece_hash     = 2;
+  orders.PieceHash             replacement_piece_hash  = 3;
+}
+
+message TransferFailed {
+  bytes piece_id = 1 [(gogoproto.customtype) = "PieceID", (gogoproto.nullable) = false];
+  enum Error {
+      NOT_FOUND                 = 0;
+      STORAGE_NODE_UNAVAILABLE  = 1;
+      UNKNOWN                   = 2;
+  }
+  Error error = 2;
+}
+
+message StorageNodeMessage {
+  oneof Message {
+    TransferSucceeded succeeded = 1;
+    TransferFailed    failed    = 2;
+  }
+}
+
+message NotReady {}
+
+message TransferPiece {
+  bytes piece_id    = 1 [(gogoproto.customtype) = "PieceID", (gogoproto.nullable) = false];; // the current piece-id
```

Comment:
this line (and a few other places that use gogoproto pieceID) has a double semi-colon at the end (I think I did this same thing in another PR copying from another proto file 😂)

Reply:
fixed